Earthfile

```diff
@@ -310,10 +310,17 @@ test-sql:
     CACHE /root/.m2
 
     COPY sql-to-dbsp-compiler sql-to-dbsp-compiler
-    RUN cd "sql-to-dbsp-compiler/SQL-compiler" && mvn package
+    RUN cd "sql-to-dbsp-compiler/SQL-compiler" && mvn package -DskipTests
     SAVE ARTIFACT sql-to-dbsp-compiler/SQL-compiler/target/sql2dbsp-jar-with-dependencies.jar sql2dbsp-jar-with-dependencies.jar
     SAVE ARTIFACT sql-to-dbsp-compiler
 
+test-sql:
```

Comment:
This will make build times worse because it won't cache the `/root/.m2`

>The CACHE command creates a cache mountpoint at <mountpoint> in the build environment. The cache mountpoint is a directory which is shared between the instances of the same build target. The contents of the cache mountpoint are preserved between builds, and can be used to share data across builds.

Unfortunately CACHE only works for the same target at the moment. But there is an issue where they plan to make sure it works across targets.

Reply:
Can we cache it twice?

Reply:
probably

Reply:
It will likely have to compile twice .. 30sec